src/main/resources/pdl/hent_ident.graphql

```diff
@@ -1,8 +1,9 @@
-query($ident: ID!, $gruppe: [IdentGruppe!], $historikk: Boolean = false) {
-    hentIdenter(ident: $ident, grupper: $gruppe, historikk: $historikk) {
+query($ident: ID!, $historikk: Boolean = false) {
+    hentIdenter(ident: $ident, historikk: $historikk) {
```

Comment:
Står forøvrig ikke dokumentert noe sted, men om man ikke spesifiserer `grupper`, så hentes alle identgrupper